examples/01_tutorials/plot_single_subject_single_run.py

```diff
@@ -268,7 +269,6 @@
 ###############################################################################
 # Report the found positions in a table
 
-from nistats.reporting import get_clusters_table
```

Comment:
No ease in tutorials, keep imports along the way: for new users, the bunch of imports at the beginning of the script is daunting and anti-pedagogic. Hence we import stuff whenever we need it.